libs/utils/utils.js

```diff
@@ -1391,7 +1391,71 @@ export function enablePersonalizationV2() {
   const enablePersV2 = getMepEnablement('personalization-v2');
   return !!enablePersV2 && isSignedOut();
 }
+export function getCookie(key) {
+  const cookie = document.cookie.split(';')
+    .map((x) => decodeURIComponent(x.trim()).split(/=(.*)/s))
+    .find(([k]) => k === key);
+  return cookie ? cookie[1] : null;
+}
+export async function determineCountry(searchParams = Object.fromEntries(PAGE_URL.searchParams)) {
```

Comment:
Hey @vgoodric , have you seen the changes introduced in https://github.com/adobecom/milo/pull/4645? `setCountry` is scheduled to run first thing, the method will set the user country in `akamai` session storage object. I don't think this is needed, we should reuse the existing code.